io-ballerina/file_string_io.bal

```diff
@@ -25,6 +26,7 @@ public isolated function fileReadString(@untainted string path) returns @tainted
 }
 
 # Reads the entire file content as a list of lines.
+# The resulting string array does not contain the terminal carriage(e.g. `\r` or `\n`).
```

Comment:
```suggestion
# The resulting string array does not contain the terminal carriage (e.g., `\r` or `\n`).
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/199/commits/bb4a9a29bfe69ed3afc9a8921e5c4627e31efb9b